### PR TITLE
Fix dev/joomla#1 - Fix list of events shown when adding an Event Info and Registration pages

### PR DESCRIPTION
**Pull request by aydun**

### CODE REVIEW

File: site/elements/civievents.php [hunk 1]
```diff
@@ -53,16 +53,13 @@ protected function getInput() {
...
     $params = array(
-      'version' => '3',
       'is_active' => 1,
-      'return.title' => 1,
-      'return.id' => 1,
-      'return.end_date' => 1,
-      'return.start_date' => 1,
-      'rowCount' => 100,
+      'return' => array("title"),
+      'start_date' => array('>=' => "today"),
```
Comment (eileenmcnaughton) [inline site/elements/civievents.php]: should this be <=

File: site/elements/civieventsonline.php [hunk 2]
```diff
@@ -30,25 +30,17 @@ protected function getInput() {
...
       'is_active' => 1,
-      'isCurrent' => 1,
-      'return.title' => 1,
-      'return.id' => 1,
-      'return.end_date' => 1,
-      'return.start_date' => 1,
-      'rowCount' => 100,
+      'is_online_registration' => 1,
+      'return' => array("title"),
+      'start_date' => array('>=' => "today"),
```
Comment (eileenmcnaughton) [inline site/elements/civieventsonline.php]: ditto here

File: site/elements/civievents.php (hunk 1 above)
Comment (aydun) [reply site/elements/civievents.php]: No, we're looking for future events so where the start date is today or later.

File: site/elements/civieventsonline.php (hunk 2 above)
Comment (aydun) [reply site/elements/civieventsonline.php]: ditto

**Comment by eileenmcnaughton** [comment]
I'm going to merge this so it hits the rc - it seems there is a clearly agreed problem & you have worked on a fix - the fix is narrowly scoped & we are likely to struggle to get another Joomla! person to review - can you ask the reporter to test the 5.4 rc?

**Comment by aydun** [comment]
Thanks for merging @eileenmcnaughton   I have asked the reporter to test via SE

**Pull request merged**